cli.go

```diff
@@ -379,7 +379,20 @@ func runConfigHooks(configs map[string]string, contrib string, current string, a
 // Execute specific hook with arguments
 // Return error message as out if error occured
 func runHook(hook string, args ...string) (status int, err error) {
-	cmd := exec.Command(hook, args...)
+	var cmd *exec.Cmd
+	// Will run a shell script with sh.exe include in Git for Windows
+	if runtime.GOOS == "windows" {
+		windowsCmd := "cmd"
+		//fmt.Println("windowsCmd is", windowsCmd)
+		cmdArgs := []string {"/C","sh.exe", hook}
```

Comment:
You assume sh.exe is under /C, I'm not familiar with Windows, but is it possible to search for that executable instead of hard code the path?

Reply:
In this case `/C` is not the drive letter, but configures `cmd.exe`'s behaviour. The relevant bit from the help screen is this one:

```
/C      Carries out the command specified by string and then terminates
```

As far as I understand, the code above should work correctly.